ivadomed/main.py

```diff
@@ -152,15 +156,16 @@ def run_main():
             transformation_dict = transform_test_params
         else:
             transformation_dict = transform_valid_params
+
+        # UNDO TRANSFORMS
+        _, undo_transforms = imed_transforms.preprare_transforms(transformation_dict.copy())
+
```

Comment:
When looking inside `prepare_transforms`: I don't see why we are calling this function here:
```python
undo_transforms = imed_transforms.UndoCompose(imed_transforms.Compose(transformation_dict))
```
as before is enough I think.
Either way I am happy. But I am missing something?